src/features/location-filters/components/FilterService.tsx

```diff
@@ -0,0 +1,57 @@
+import { Filter } from "@/components/Filter/Filter";
+import { FilterHeader } from "@/components/Filter/FilterHeader";
+import { useTranslation } from "next-i18next";
+import { FilterControl } from "@/components/Filter/FilterControl";
+import { MenuItem } from "@mui/material";
+import {
+  useServiceFilter,
+  filters as serviceFilters,
+  ServiceCategory,
+} from "../stores/useServiceFilter";
+
+export const FilterService = () => {
+  const { t } = useTranslation("home");
+  const { isOpen, actions, selectedCategories } = useServiceFilter();
+
+  console.log({ selectedCategories });
```

Comment:
?

Reply:
It is for debugging purposes, will be cleaned up later.

Reply:
> i asked him to open a pr so i could use what he did as ref, no worries.

lol draft PR is a feature for this usecase, ok

Reply:
oh i keep forgetting that github has a draft feature now man @eraygundogmus sorrryyyyyy mybad.